databases/week2/homework_part2.sql

```diff
@@ -0,0 +1,32 @@
+-- Create a new database containing the following tables:
+CREATE DATABASE school_db;
+DEFAULT CHARACTER SET = "utf8mb4";
+USE school_db;
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+CREATE Table Class (
```

Comment:
very well specified table, good use of primary key

Reply:
Thanks